osu.Game.Rulesets.Osu/UI/Cursor/OsuCursor.cs

```diff
@@ -37,17 +37,19 @@ protected override void SkinChanged(ISkinSource skin, bool allowFallback)
         [BackgroundDependencyLoader]
         private void load()
         {
-            InternalChild = expandTarget = new Container
+            SkinnableDrawable cursor;
+            InternalChild = new Container
             {
                 RelativeSizeAxes = Axes.Both,
                 Origin = Anchor.Centre,
                 Anchor = Anchor.Centre,
-                Child = new SkinnableDrawable(new OsuSkinComponent(OsuSkinComponents.Cursor), _ => new DefaultCursor(), confineMode: ConfineMode.NoScaling)
+                Child = cursor = new SkinnableDrawable(new OsuSkinComponent(OsuSkinComponents.Cursor), _ => new DefaultCursor(), confineMode: ConfineMode.NoScaling)
                 {
                     Origin = Anchor.Centre,
                     Anchor = Anchor.Centre,
                 }
             };
+            cursorSprite = cursor.Drawable as OsuCursorSprite;
```

Comment:
make this a property instead. something like

`private Drawable expandTarget => (cursor.Drawable as OsuCursorSprite)?.ExpandTarget ?? cursor;`

to allow for fallback in case a different type is returned